src/executor/admin/SpaceExecutor.cpp

```diff
@@ -144,15 +150,17 @@ folly::Future<Status> ShowCreateSpaceExecutor::execute() {
                 Row row;
                 row.values.emplace_back(properties.get_space_name());
                 auto fmt = "CREATE SPACE `%s` (partition_num = %d, replica_factor = %d, "
-                           "charset = %s, collate = %s, vid_type = %s)";
+                           "charset = %s, collate = %s, vid_type = %s) ON %s";
                 row.values.emplace_back(folly::stringPrintf(
                     fmt,
                     properties.get_space_name().c_str(),
                     properties.get_partition_num(),
                     properties.get_replica_factor(),
                     properties.get_charset_name().c_str(),
                     properties.get_collate_name().c_str(),
-                    SchemaUtil::typeToString(properties.get_vid_type()).c_str()));
+                    SchemaUtil::typeToString(properties.get_vid_type()).c_str(),
+                    properties.__isset.group_name ? properties.get_group_name()
+                                                  : new std::string("default")));
```

Comment:
It seems that you are trying to format a `std::string` like a C string? i.e. `%s`

Reply:
Setting the default value with `default` and it seems should be a `std::string*`

Reply:
Why not use `"default"` directly? It seems a memory leak.